### DISCUSSION TITLE
Create and manage custom services with GUI

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferdium/ferdium-app#1699] [Ideas]
Please add the ability to create custom services within the app with a GUI and to be able to manage custom services as well within the app,

**Comment by vraravam** [comment]
This is already possible via the "Add Service" UI

**Reply by trymeouteh** [reply]
Are custom services like JSON files you download from the web and place in a specific directory on your computer which are made by 3rd party devs?

What is the difference between custom services and custom website?

**Reply by vraravam** [reply]
Please try it out. imo, they are the same
